### PR TITLE
#64  implements vite setup

### PR DESCRIPTION
**Pull request by 94726**
This PR replaces the setup initially created by `create-react-app` with `Vite`
You should definetly checkout this PR and see the speed yourself.

### CODE REVIEW

**Review by quintus-luis — APPROVED** [review]

**Review by LeoTuet — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -25,14 +33,9 @@
     "react-scripts": "5.0.0",
```
Comment (Aloso) [inline package.json:33]: `react-scripts` is a dependency of `create-react-app`. I think if we use Vite, we should remove this.

File: package.json [hunk 2]
```diff
@@ -2,6 +2,13 @@
   "name": "ui-we-want-to-sleep",
   "version": "0.1.0",
   "private": true,
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "test": "react-scripts test",
```
Comment (Aloso) [inline package.json:8]: This should be replaced with `jest`. How to test React/TS apps with Vite is explained [here](https://codingwithmanny.medium.com/quick-jest-setup-with-vitejs-react-typescript-82f325e4323f), for example.

**Review by Aloso — CHANGES_REQUESTED** [review]

**Comment by Aloso** [comment]
@94726 Could you please also remove the line from the Dockerfile that installs `react-scripts`? Then I think this is ready to merge.

**Comment by 94726** [comment]
> @94726 Could you please also remove the line from the Dockerfile that installs `react-scripts`? Then I think this is ready to merge.

Oh sorry, didn't see it. Wonder why it was added in the first place 😕.

Gonne change it later.

**Review by Aloso — APPROVED** [review]
LGTM

**Pull request merged**